packages/builders/src/messages/poll/Poll.ts

```diff
@@ -0,0 +1,227 @@
+import type { JSONEncodable } from '@discordjs/util';
+import type { RESTAPIPoll, APIPollMedia } from 'discord-api-types/v10';
+import { PollLayoutType } from 'discord-api-types/v10';
+import { normalizeArray, type RestOrArray } from '../../util/normalizeArray.js';
+import { resolveBuilder } from '../../util/resolveBuilder.js';
+import { isValidationEnabled } from '../../util/validation.js';
+import { pollPredicate } from './Assertions';
+import { PollAnswerBuilder } from './PollAnswer.js';
+import { PollQuestionBuilder } from './PollQuestion.js';
+
+export interface PollBuilderData extends Omit<RESTAPIPoll, 'answers' | 'question'> {
```

Comment:
Not the naming convention we go for

```suggestion
export interface PollData extends Omit<RESTAPIPoll, 'answers' | 'question'> {
```

Reply:
Couldn't the same be said for `EmbedBuilderData` in this case then? Just to understand why that has "Builder" in its name but this interface shouldn't.